### PR TITLE
Add statusphere content to the site

### PR DESCRIPTION
**Pull request by PierrickGT**
Issue: https://github.com/status-im/status.im/issues/136

### CODE REVIEW

File: themes/navy/layout/index.swig [hunk 1]
```diff
@@ -128,7 +127,7 @@
     </div>
 
     <div class="cards cards--projects cards--three">
-        <div class="card card--app">
```
Comment (andytudhope) [inline themes/navy/layout/index.swig]: Why did you remove these classes? Surely that will remove the colours they're supposed to turn to on hover, or on mobile screens? Or am I missing something in my too-hasty review?

Comment (PierrickGT) [reply themes/navy/layout/index.swig]: No, I've used `nth-child()` in order to have colors dynamically, no matter the number of items.
The colors repeat every 6 items. It was easier to use CSS for Statusphere instead of using JS to add a specific class.
